### PR TITLE
Update dependencies

### LINKED ISSUE #239: npm install error
When running npm install an error is thrown saying it cannot find event-stream@3.3.6. This version has been removed from the npm registry because it contains a malicious package.

### PR DESCRIPTION
**Pull request by alub**
All dependencies are updated. This fixes #239 and also the following:

```
node-pre-gyp WARN Using request for node-pre-gyp https download 
node-pre-gyp WARN Tried to download(404): https://node-precompiled-binaries.grpc.io/grpc/v1.14.2/node-v67-linux-x64-glibc.tar.gz 
node-pre-gyp WARN Pre-built binaries not found for grpc@1.14.2 and node@11.3.0 (node-v67 ABI, glibc) (falling back to source compile with node-gyp) 
node-pre-gyp WARN Pre-built binaries not installable for grpc@1.14.2 and node@11.3.0 (node-v67 ABI, glibc) (falling back to source compile with node-gyp) 
node-pre-gyp WARN Hit error Connection closed while downloading tarball file 
```

### CODE REVIEW

**Comment by samuelli** [comment]
Hey @alub, can you sign the CLA please?

**Comment by alub** [comment]
I signed it!

**Comment by alub** [comment]
I’ve signed a corporate CLA, but I haven’t received an answer yet.

**Comment by alub** [comment]
I signed it!

File: package.json [hunk 1]
```diff
@@ -25,36 +25,36 @@
   "dependencies": {
     "@google-cloud/datastore": "^1.4.2",
     "@types/fs-extra": "^5.0.4",
-    "@webcomponents/webcomponentsjs": "^1.2.6",
-    "chrome-launcher": "^0.10.2",
+    "@webcomponents/webcomponentsjs": "^1.3.0",
```
[the remark this answers is not in the document]
Comment (alub) [reply package.json]: I wouldn’t know how to test this change. Can I suggest you do it in a separate PR?

**Pull request closed without merging**